cli/azd/pkg/input/console.go

```diff
@@ -214,7 +214,7 @@ func (c *AskerConsole) Message(ctx context.Context, message string) {
 			panic(fmt.Sprintf("Message: unexpected error during marshaling for a valid object: %v", err))
 		}
 		fmt.Fprintln(c.writer, string(jsonMessage))
-	} else if c.formatter == nil || c.formatter.Kind() == output.NoneFormat {
+	} else if c.formatter != nil {
```

Comment:
Did you figure out what was going on here?  I'm worried that this will causes messages to be written when we are using the table formatter and that will break weird things.

I think you had added this when debugging the test failure where the error message was being written to stdout instead of stderr, but it's also not clear how that would have fixed it (the second part of the diff here would fix it and makes sense to me if we want to accept the new behavior).  Do we need to roll this part back?

Reply:
@ellismg I would consider this a bug.  I don't know of a use case where we would not want the message printed out unless there is no formatter specified.  In a command that is even using a table formatter we want one line / error message to print out to std out.  

If you know of a use case that deviates from this we can work through it.